rllib/agents/dqn/dqn_policy.py

```diff
@@ -230,10 +202,9 @@ def build_q_model(policy, obs_space, action_space, config):
     return policy.q_model
 
 
-def build_q_networks(policy, q_model, input_dict, obs_space, action_space,
-                     config):
-
-    # Action Q network
+def action_sampler_function(policy, q_model, input_dict, obs_space,
```

Comment:
Consider `sample_action_from_q_network`.

Reply:
:+1: Changed also in simple_q